packages/dx-react-grid-demos/src/demo-sources/grid-virtual-scrolling/stretching-to-parent-element.jsxt

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Grid,
+  VirtualTable,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-<%&themeName%>';
+
+import {
+  generateRows,
+  defaultColumnValues,
+} from '../../../demo-data/generator';
+
+const getRowId = row => row.id;
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { name: 'id', title: 'ID' },
+        { name: 'name', title: 'Name' },
+        { name: 'sex', title: 'Sex' },
+        { name: 'city', title: 'City' },
+        { name: 'car', title: 'Car' },
+      ],
+      rows: generateRows({
+        columnValues: { id: ({ index }) => index, ...defaultColumnValues },
+        length: 100000,
+      }),
+    };
+  }
+
+  render() {
+    const { rows, columns } = this.state;
+
+    return (
+      <<%&wrapperTag%> style={{ height: '600px' }}>
+        <Grid
+          rows={rows}
```

Comment:
Maybe add to this demo slider with a range of values [100-500]. This slider value will be a change a parent `height`.
Or simply change the parent height to 400px or low, because 600 and 570 look the same.